docs/src/gs-configure-elastiservice.md

```
@@ -0,0 +1,122 @@
+# Configure ElastiService
+
+To enable scale to 0 on any deployment, we will need to create an ElastiService custom resource for that deployment. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)